### PR TITLE
Add env name to logs

### PR DESCRIPTION
**Pull request by pr33thi**
- Env name gets added to the log locations
- Replaced `{}` with `<>` so that `.format()` doesn't get confused

In reference to this: https://github.com/dimagi/commcare-cloud/pull/2033#issuecomment-414816646

### CODE REVIEW

File: src/commcare_cloud/commands/ansible/service.py [hunk 1]
```diff
@@ -59,12 +59,12 @@ def __init__(self, environment, ansible_context):
...
         self.ansible_context = ansible_context
 
-    def run(self, action, host_pattern=None, process_pattern=None):
+    def run(self, action, env_name, host_pattern=None, process_pattern=None):
         if action == 'help':
             self.print_help()
             return 0
         elif action == 'logs':
-            print("Logs can be found at:\n{}".format(self.log_location))
+            print("Logs can be found at:\n{}".format(self.log_location.format(env_name)))
```
Comment (dannyroberts) [inline src/commcare_cloud/commands/ansible/service.py]: nice. You could also keep it as `{env}` below and use `.format(env=env_name)` here. That way someone reading just that line (or producing a similar one later) knows just by looking at it what's intended.

**Comment by dannyroberts** [comment]
And +1 on changing {} to <>, that seems like a good solution.

File: src/commcare_cloud/commands/ansible/service.py (hunk 1 above)
Comment (pr33thi) [reply src/commcare_cloud/commands/ansible/service.py]: Cool, didn't realize I could do that; https://github.com/dimagi/commcare-cloud/pull/2055/commits/90509753db851fc32509115dd32791c1087c2c19

File: src/commcare_cloud/commands/ansible/service.py [hunk 2]
```diff
@@ -289,14 +289,14 @@ class ElasticsearchClassic(AnsibleService):
     name = 'elasticsearch-classic'
     service_name = 'elasticsearch'
     inventory_groups = ['elasticsearch']
-    log_location = '/opt/data/{ecrypt}/elasticsearch-1.7.3/logs'
+    log_location = '/opt/data/<ecrypt>/elasticsearch-<version>/logs'
```
[the remark this answers is not in the document]
Comment (pr33thi) [reply src/commcare_cloud/commands/ansible/service.py]: https://github.com/dimagi/commcare-cloud/pull/2055/commits/0c88d01f1a2d486b1bd61f5e4c303c692170f864

File: src/commcare_cloud/commands/ansible/service.py [hunk 3]
```diff
@@ -59,12 +59,12 @@ def __init__(self, environment, ansible_context):
         self.environment = environment
         self.ansible_context = ansible_context
 
-    def run(self, action, host_pattern=None, process_pattern=None):
+    def run(self, action, env_name, host_pattern=None, process_pattern=None):
```
[the remark this answers is not in the document]
Comment (pr33thi) [reply src/commcare_cloud/commands/ansible/service.py]: https://github.com/dimagi/commcare-cloud/pull/2055/commits/ba77354cf46970dac50e5c12f8f5db379cbbb35a

**Pull request merged**